jenkins/ci_build_win_desktop.ps1

```diff
@@ -57,7 +62,7 @@ function Build() {
     New-Item -ItemType Directory -ErrorAction Ignore $directory
     Push-Location $directory
 
-    & "C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 15 2017" -A x64 -DEDITION="$Edition" -DCMAKE_INSTALL_PREFIX="$pwd/libcblite-${Version}" ..
+    & "C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 15 2017" -A "${Architecture}" -DEDITION="$Edition" -DCMAKE_INSTALL_PREFIX="$pwd/libcblite-${Version}" ..
```

Comment:
Just as a comment not relevant to this PR, you might want to stop specifying a specific visual studio.  This one is quite old now and really should be 2019 or 2022.  If you leave it out then CMake will choose the latest one it can find.

Reply:
I will test it out.